sdk/notificationhubs/notification-hubs/package.json

```
@@ -140,15 +140,16 @@
     "util": "^0.12.4"
   },
   "dependencies": {
+    "@azure/core-auth": "~1.4.1",
```

We can use hat right?
```suggestion
    "@azure/core-auth": "^1.4.1",
```